LLK/scratch-www
# Fix R2-4470: Only allows birthdays >= 13 yrs old for teacher reg

Helps fix https://github.com/LLK/scratchr2/issues/4470 from the UX side of things by preventing moving forward with teacher registration if the birthday given is less than 13.

### Test Cases ###
* Try to register with a birthday less than 13 years before the current date (could be 12 years, 11 months, even). You should get an error bubble
* Try to register with a birthday >= 13 years before current date. You should be able to proceed.
